README.md

```diff
@@ -33,6 +33,7 @@
   - Gaussian blur
   - Horizontal blur
   - Padding
+  - Rotation
```

Comment:
This differs from the method/module name. I think it would be great to have them identical.

Reply:
That's a good observation! What do you think the appropriate naming convention should be for operation?

I've been typically using the verb form of words (`pad`, `rotate`) as the name of the method to _do_ stuff with. In lists like this, it reads better when it's the noun form of the word (`padding`, `rotation`), and with the desire have these two names being the same, it leads to method names taking the noun form.

I'm good renaming the method to the noun form, I just this realization now and wanted to share, to see if my own observation matches reality.

Reply:
Unless you think `rotate` (Rotate) is the better word to use everywhere?

Reply:
The noun form seems good. I don't have a strong preference, I just want it to be consistent. The verb form would also look weird in `README.md`.